internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -28,10 +28,11 @@
 {{- end}}
 {{- if eq .WorkloadType "Load Balanced Web Service"}}
   PortMappings:
-    - ContainerPort: !Ref ContainerPort
-    {{/* Multiple exposed port is not supported yet. Therefore, if the target container is the sidecar, the target port must be the one and only one port that the container exposes. */}}
-    {{- if eq .HTTPTargetContainer.Name .WorkloadName}}
-      Name: target
+    {{- range $portMapping := .PortMappings}}
+    - ContainerPort: {{$portMapping.ContainerPort}}
+    {{- if eq $.HTTPTargetContainer.Name $.WorkloadName}}
```

Comment:
Great question Wan: for now until we don't support multiple ports this is true because HTTPTargetContainer is the only container who is exposing port that too only one port. But when multiple ports come into the picture then we will remove this condition as HTTPTargetContainer is not the only container who is exposing the ports. does that make sense?